packages/opentelemetry/src/spanExporter.ts

```diff
@@ -192,6 +189,7 @@ export class SentrySpanExporter {
       );
 
     for (const span of sentSpans) {
+      this._sentSpans.set(span.spanContext().spanId, Date.now() + DEFAULT_TIMEOUT * 1000);
```

Comment:
Just noticed, we should probably move out the `Date.now() + DEFAULT_TIMEOUT * 1000` part to avoid recalculating it over the loop.

Reply:
ah, great catch too! 👍 will do!